chart/templates/deployment.yaml

```diff
@@ -0,0 +1,151 @@
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          livenessProbe:
+            exec:
+              command:
+                - "true"
+          readinessProbe:
+            exec:
+              command:
+                - "true"
+          env:
+            {{- if .Values.config.awsAccessKeyID }}
+            - name: "AWS_ACCESS_KEY_ID"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: awsAccessKeyID
+            {{- end }}
+            {{- if .Values.config.awsRegion }}
+            - name: "AWS_REGION"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: awsRegion
+            {{- end }}
+            {{- if .Values.config.awsSecretAccessKey }}
+            - name: "AWS_SECRET_ACCESS_KEY"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: awsSecretAccessKey
+            {{- end }}
+            {{- if .Values.config.baaahThreadiness }}
+            - name: "BAAAH_THREADINESS"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: baaahThreadiness
+            {{- end }}
+            {{- if .Values.config.githubAuthToken }}
+            - name: "GITHUB_AUTH_TOKEN"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: githubAuthToken
+            {{- end }}
+            {{- if .Values.config.obotServerAuthAdminEmails }}
+            - name: "OBOT_SERVER_AUTH_ADMIN_EMAILS"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: obotServerAuthAdminEmails
+            {{- end }}
+            {{- if .Values.config.obotServerAuthClientID }}
+            - name: "OBOT_SERVER_AUTH_CLIENT_ID"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: obotServerAuthClientID
+            {{- end }}
+            {{- if .Values.config.obotServerAuthClientSecret }}
+            - name: "OBOT_SERVER_AUTH_CLIENT_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: obotServerAuthClientSecret
+            {{- end }}
+            {{- if .Values.config.obotServerAuthCookieSecret }}
+            - name: "OBOT_SERVER_AUTH_COOKIE_SECRET"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "obot.config.secretName" . }}
+                  key: obotServerAuthCookieSecret
+            {{- end }}
```

Comment:
These three secrets can be removed. With my auth provider refactoring, these are no longer provided to the obot server, and instead are provided when configuring an auth provider.

Reply:
We would still need the cookie secret, wouldn't we? That's not directly related to the auth provider

Reply:
Nope, those get automatically generated and stored for each auth provider internally. They are no longer able to be provided by the user.